nucypher/blockchain/eth/signers.py

```diff
@@ -160,6 +171,8 @@ def sign_transaction(self, transaction_dict: dict) -> HexBytes:
 
 class ClefSigner(Signer):
 
+    URI_SCHEME = 'keystore://'
```

Comment:
Should ClefSigner be `keystore://` or `clef://`?

Reply:
:heavy_check_mark: